pipeline/backend/common/script_posix.go

```diff
@@ -53,10 +59,12 @@ fi
 unset CI_NETRC_USERNAME
 unset CI_NETRC_PASSWORD
 unset CI_SCRIPT
-mkdir -p "$CI_WORKSPACE"
-cd "$CI_WORKSPACE"
+mkdir -p "{{.WorkDir}}"
```

Comment:
this could probably be left as it was

Reply:
or could be both just in case they do not match

Reply:
no that's totaly unrelated - and normaly the clone plugin is in charge of makeing it exist